__tests__/commands/FQL10Collections.test.js

```diff
@@ -115,6 +116,118 @@ describe("FQL 10 Collections", () => {
       await verifyNoCollections();
     });
 
+    it("manages a collection with ttl_days", async () => {
+      const config = {
+        collections: {
+          MyNewCollection: {
+            ttl_days: 10,
+          },
+        },
+      };
+
+      // Run create
+      await runDeploy(config);
+      let logs = [
+        "FQL v10 schema update in progress...",
+        "Collection: MyNewCollection created",
+        "FQL v10 schema update complete",
+      ];
+      verifyLogs(log, logs);
+      await verifyCollections(client, config.collections);
```

Comment:
does this actually check that `ttl_days` is set on the created collection?

Reply:
nice catch - should be fixed now